src/Service.Tests/Multidab-config.CosmosDb_NoSql.json

```diff
@@ -0,0 +1,379 @@
+{
+  "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
+  "data-source": {
```

Comment:
can you clarify what makes these `multidab-config` files different from the already present ones in the test project? it doesn't look like any multi-datasource specific changes are made to these files.

Reply:
All the existing files have duplicate entities across them. these ones have unique entities across them.